website/content/docs/proposals/proposal-002-run.md

```diff
@@ -0,0 +1,223 @@
+with context. It is really helpful to provide a "source of truth" for the work
+so that people aren't searching all over the place for lost context.
+-->
+
+* [Slack discussion on benchmark test framework](https://cloud-native.slack.com/archives/C060EDHN431/p1708416918423089?thread_ts=1708348336.259699&cid=C060EDHN431)
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is:
+outlining your solution to the problem described in the Motivation section.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this proposal is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+
+* As a CNCF Project Maintainer,
+
+* As a Green Review WG reviewer,
+
+* As a Green Review WG cluster maintainer,
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate?
+Think broadly.  For example, consider how this will impact or be impacted
+by scaling to support more CNCF Projects.
+
+How will this affect the benchmark tests, CNCF Project Maintainers, pipeline maintainers, etc?
+-->
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include manifests or workflow examples
+(though not always required) or even code snippets. If there's any ambiguity
+about HOW your proposal will be implemented, this is the place to discuss them.
+-->
+
+Workflows can be fetched from other GitHuborganizations and repositories using the `jobs.<job_id>.uses` syntax defined here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses
+
+```
+jobs:
+  call-workflow-in-another-repo: # job_id
+    uses: octo-org/another-repo/.github/workflows/workflow.yml@v1 # refer to self or other repo
+```
+
+This syntax can be configured to use `@main` or `@another-branch` which would be nice for versioning and testing specific releases.
+
+This means there is no need to authenticate to any repository as long as the external repositories are public.
+
+How will the job run in the cluster? We assume that the workflow already contains a kubeconfig to authenticate with the test cluster and Falco has already been deployed to it.
```

Comment:
Indeed, check this line of the PR: https://github.com/cncf-tags/green-reviews-tooling/pull/88/files#diff-67a930dd3c23a2ba82d45080a474a30c505f1a631e13c712d38d6faa2ba4160cR195